art/estimators/classification/pytorch.py

```diff
@@ -63,6 +63,9 @@ def __init__(
         input_shape: Tuple[int, ...],
         nb_classes: int,
         optimizer: Optional["torch.optim.Optimizer"] = None,  # type: ignore
+        use_amp: bool = False,
+        opt_level: str = "O1",
+        loss_scale: Optional[Union[float, str]] = "dynamic",
```

Comment:
In the ASR estimator and attack, a value of `1.0` is used. I initially didn't understand why strings were allowed. Maybe you could add the possible string arguments to the docstring?

Reply:
Thank you @hesseltuinhof , this fixed.